### ISSUE TITLE
Solus fails to install only if full disk encryption is enabled

### ISSUE THREAD

**Issue by n1v3k** [getsolus/packages#4317]
### Please confirm there isn't an existing open bug report

- [X] I have searched open bugs for this issue

### Summary

Good evening everyone, so I have this MacBook Air 2014, i5 4260U, 8GB/512GB and installing Solus on it with full disk encryption fails, here is the log https://termbin.com/gatx2 [gatx2.txt](https://github.com/user-attachments/files/17689700/gatx2.txt) Ubuntu 22.04.1, openSUSE Leap 15.6 and Tumbleweed install just fine all of them using full disk encryption.

Solus installs fine with no encryption but I need full disk encryption. What could it be?

### Steps to reproduce

Just go with normal installation with full disk encryption and eventually will take some time until the error appears.

### Expected result

To complete the installation of Solus.

### Actual result

Failed to install, log is attached.

### Environment

- [X] Is system up to date?

### Repo

Shannon (stable)

### Desktop Environment

Plasma

### System details

```
live@solus ~ $ inxi -b
System:
  Host: solus Kernel: 6.10.13-304.current arch: x86_64 bits: 64
  Desktop: KDE Plasma v: 6.1.5 Distro: Solus 4.6 convergence
Machine:
  Type: Laptop System: Apple product: MacBookAir6,1 v: 1.0
    serial: <superuser required>
  Mobo: Apple model: Mac-35C1E88140C3E6CF v: MacBookAir6,1
    serial: <superuser required> UEFI: Apple v: 474.0.0.0.0 date: 08/23/2022
Battery:
  ID-1: BAT0 charge: 33.8 Wh (100.0%) condition: 33.8/38.8 Wh (87.3%)
CPU:
  Info: dual core Intel Core i5-4260U [MT MCP] speed (MHz): avg: 800
    min/max: 800/2700
Graphics:
  Device-1: Intel Haswell-ULT Integrated Graphics driver: i915 v: kernel
  Display: wayland server: X.org v: 1.21.1.13 with: Xwayland v: 24.1.3
    compositor: kwin_wayland driver: N/A resolution: 1366x768
  API: OpenGL v: 4.6 compat-v: 4.5 vendor: intel mesa v: 24.2.3
    renderer: Mesa Intel HD Graphics 5000 (HSW GT3)
Network:
  Device-1: Broadcom BCM4360 802.11ac Dual Band Wireless Network Adapter
    driver: bcma-pci-bridge
Drives:
  Local Storage: total: 581.15 GiB used: 0 KiB (0.0%)
Info:
  Memory: total: 8 GiB available: 7.7 GiB used: 1.23 GiB (16.0%)
  Processes: 189 Uptime: 0m Shell: Bash inxi: 3.3.36
live@solus ~ $

```

### Other comments

A big hug for anyone who can help me :)

**Comment by riffer33** [comment]
I have a similar issue on a Thinkpad X1 Carbon Gen 8 using the latest Solus-GNOME-Release-2024-10-14.iso

The boot hangs after the install.  Examination of the disk indicates Solus has been installed.

After reading this bug report I wiped the disk and installed Solus without disk encryption and I was able to boot into solus.

**Comment by davidjharder** [comment]
I may be experiencing something similar with a new laptop I have: With full disk encryption, the install completes, but the machine hangs on the vendor splash screen on reboot. Discovered this splash screen is hiding the expected LUKS prompt. If I blindly enter the encryption password, boot continues normally

**Comment by TraceyC77** [comment]
@clintre , does this sound like what you were seeing?

**Comment by clintre** [comment]
Yes, this sounds similar, and I was thinking the same originally.

I did turn off the vendor full-screen logo, fastboot, etc. That did not work.

So I set a 5-second time limit via clr-boot-manager. So I get to the systemd boot and see it show before it tries to boot, and it still does not show the luks prompt. 

This only happens dedicated Nvidia. On laptops with hybrid it will get past it, but if I set that to dedicated only it too has the same issue. Not sure if it would happen with grub.

When I added `nvidia_drm.modeset=1 nvidia_drm.fbdev=1 `to the boot, it would show the LUKS prompt, despite still being on the nouveau driver. Once nvidia driver is installed this all goes away.

**Comment by TraceyC77** [comment]
Reading this over more closely, there are two distinct, separate issues reported here.

1. The original report is about there being an error and the install failing when full disk encryption is chosen  
@n1v3k , are you still able to reproduce this? If so, can you please attach your logs to this report? The link to your logs is now 404.

2. With full disk encryption, after a *successful* installation, the LUKS password prompt does not appear (but blindly typing in the password will allow decryption) - I've opened up a new bug for this  
https://github.com/getsolus/packages/issues/8574
